### ISSUE TITLE
Async Storage will be removed from react-native core

### ISSUE THREAD

**Issue by PerraiD** [vmurin/react-native-azure-auth#22]
Hi, i'm working on building react-native app with your great library.
There is a warning about async storage that will be remove. 
react-native version : 0.59.5

```
Warning: Async Storage has been extracted from react-native core and will be removed in a future release. It can now be installed and imported from '@react-native-community/async-storage' instead of 'react-native'. See https://github.com/react-native-community/react-native-async-storage
```

thanks.

**Comment by vmurin** [comment]
Thanks for the hint. I'll take care about it.

**Comment by vmurin** [comment]
The issue solved in release v1.1.0 (91f2d41)
